### ISSUE TITLE
Preopens & limitations

### ISSUE THREAD

**Issue by zifeo** [kateinoigakukun/wasi-vfs#17]
Great package! Two short questions:

1. I understand the [limitations in the readme](https://github.com/kateinoigakukun/wasi-vfs#limitations) should be covered now. Is there any other known limitations?
2. Is this approach compatible with preopens from wasi (e.g. pack with vfs, then share an additional directory at wasi init)?

**Comment by andrewmd5** [comment]
Hey @kateinoigakukun also curious about this. Are we able to pack one directory but still pass through to the WASI implementation for a preopen?

**Comment by kateinoigakukun** [comment]
Yes, access to non-prepackaged directories is falling back to host wasi syscalls
